source/features/warn-pr-from-master.tsx

```diff
@@ -38,3 +43,11 @@ void features.add(import.meta.url, {
 	deduplicate: 'has-rgh',
```

Comment:
Both of these can be dropped.

Also, is the banner removed/hidden when the user selects another branch?